tests/Samples.Server.Tests/BaseTest.cs

```diff
@@ -39,10 +58,18 @@ public virtual void Dispose()
         {
             Client.Dispose();
             Server.Dispose();
+
+#if !NETCOREAPP2_2
+            Host.Dispose();
+#endif
         }
 
         protected TestServer Server { get; }
 
         protected HttpClient Client { get; }
+
+#if !NETCOREAPP2_2
+        protected IHost Host { get; }
```

Comment:
Protected? Any usages?

Reply:
Nope, neither for the two above. All private now.